dags/copy_deduplicate.py

```diff
@@ -324,7 +261,15 @@
         email=["telemetry-alerts@mozilla.com", "jklukas@mozilla.com"],
     )
 
-    firefox_nondesktop_exact_mau28 >> simpleprophet_forecasts_mobile
+    wait_for_firefox_nondesktop_exact_mau28 = ExternalTaskSensor(
+        task_id="wait_for_firefox_nondesktop_exact_mau28",
+        external_dag_id="bqetl_nondesktop",
+        external_task_id="telemetry__firefox_nondesktop_exact_mau28_raw__v1",
+        check_existence=True,
+        dag=dag,
+    )
+
+    wait_for_firefox_nondesktop_exact_mau28 >> simpleprophet_forecasts_mobile
```

Comment:
I'm concerned about the fact that we now have the nondesktop DAG waiting on `copy_deduplicate`, and then  this later task in `copy_deduplicate` that depends on the nondesktop DAG.

I think the `simpleprophet_forecasts_mobile` task should rightfully move into the `kpi_dashboard` DAG, as the forecasts exist primarily for the KPI dash. 

But we're working toward generating that one via bq-etl as well, which would mean we can't put this task there.

But also the kpi dash DAG contains a smoot-related query that's only there because it had similar dependencies. Really, we should probably move all the smoot queries to a `bqetl_gud` DAG that's generated.

So.... that's a lot of potential for moving thing around. Out of scope for this PR, certainly. But I think the above steps would all be good ones to take in follow-ups. The KPI dash DAG should probably remain hand-written; all the forecast jobs (desktop, nondesktop, and fxa) would live there, and then it would perform just the one query for materializing the kpi_dashboard table.

Reply:
Yeah moving things around and having a DAG for GUD related queries seems reasonable. I'll open a follow-up PR.